package.json

```diff
@@ -116,13 +116,13 @@
   },
   "dependencies": {
     "@simonwep/selection-js": "^1.6.0",
```

Comment:
@au-re This package is deprecated. The author has changed the name. https://github.com/Simonwep/selection 
2.x doesn't work with Ramen.